### PR TITLE
Add information on how to renew certificate

### PR DESCRIPTION
**Pull request by baptistemesta**
Closes [ICI-1130](https://bonitasoft.atlassian.net/browse/ICI-1130)

### CODE REVIEW

File: md/installation_guide.md [hunk 1]
```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
 
 Then when using the installer provide this generated file as a keystore, the same password for the keystore and the key, and the same alias.
 
+#### Renew certificate
```
Comment (NathalieC) [inline md/installation_guide.md]:
```suggestion
#### Update the certificate
```

File: md/installation_guide.md [hunk 2]
```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
...
+
+When the certificate expires, it can be renewed by reinstalling the backend (not the storage).
```
Comment (NathalieC) [inline md/installation_guide.md]: renewing needs to be done on LetsEncrypt (for example), right? It looks like Bonita can renew the certificate.

Comment (NathalieC) [inline md/installation_guide.md]:
```suggestion
When the certificate expires, renew it on the cerbot of your choice.
To use if on BICI, you need to reinstall the backend (not the storage).
```

File: md/installation_guide.md [hunk 3]
```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
...
+
```
Comment (NathalieC) [inline md/installation_guide.md:100]:
```suggestion
To do so:
```

File: md/installation_guide.md [hunk 4]
```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
...
+* stop the backend using `bonita-ici stopApp`
+* start and deploy the backend using either the new certificate or newly generated one using `bonita-ici startApp deploy`
```
Comment (NathalieC) [inline md/installation_guide.md:103]: What is the difference between "new certificate" and "newly generated one"?
We can talk maybe. I'm missing something... :/

File: md/installation_guide.md [hunk 5]
```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
...
+
+If the certificate is autogenerated, the Bonita platform JVM needs to be restarted.
+
+
 ### Stop the module
```
Comment (NathalieC) [inline md/installation_guide.md:108]: We talk about the stopping the add-on for updating the certificate before we tell Patty how to stop the add-on.
Maybe updating should come after stopping?
Wording: "Stop the Add-on"

**Review by NathalieC — CHANGES_REQUESTED** [review]
Comment may come from my lack of understanding on https certificates. Let's talk if needed.

File: md/installation_guide.md (hunk 2 above)
Comment (baptistemesta) [reply md/installation_guide.md]: ok, i'll change that

File: md/installation_guide.md (hunk 5 above)
Comment (baptistemesta) [reply md/installation_guide.md:108]: yep maybe

File: md/installation_guide.md (hunk 4 above)
Comment (baptistemesta) [reply md/installation_guide.md:103]: `generated one` is the one that the installer generate if you choose to do so, and `new certificate` if you provided one

File: md/installation_guide.md (hunk 5 above)
Comment (baptistemesta) [reply md/installation_guide.md:108]: in fact its not the same title section +  I prefere  keeping "update the certificate" close to "security configuration"

Comment (NathalieC) [reply md/installation_guide.md:108]: ok

File: md/installation_guide.md (hunk 4 above)
Comment (NathalieC) [reply md/installation_guide.md:103]: ok. I think it can be clarified, but again, not a big deal.

**Pull request merged**